lib/rouge/lexers/raml.rb

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*- #
+
+module Rouge
+  module Lexers
+    load_lexer 'yaml.rb'
+    
+    class RAML < YAML
+      title "RAML"
+      desc "Everyday I'm RAML-ing"
```

Comment:
This made me laugh but does RAML have an official website. This is usually something we want to include here. Here's an example from the Ruby lexer:

https://github.com/rouge-ruby/rouge/blob/1c6df45752f4e0a2a69f3b0888fada3b782d69c2/lib/rouge/lexers/ruby.rb#L8